molo/core/content_import/validation.py

```diff
@@ -0,0 +1,106 @@
+from babel import Locale
+
+from molo.core.models import SiteLanguage
+
+from unicore.content.models import Category, Page
+
+
+class ContentImportValidation(object):
+    errors = []
+
+    def __init__(self, ws):
+        self.ws = ws
+
+    def is_validate_for(self, locales):
+        self.errors = []
+
+        if self.validate_wagtail_has_no_language():
+            for l in locales:
+                self.validate_translated_content_has_source(l)
+                self.validate_page_primary_category_exists(l)
+                self.validate_translated_content_source_exists(l)
+
+        return self.errors
+
+    def validate_wagtail_has_no_language(self):
```

Comment:
What are you trying to test here - you're looping through all the languages and throwing an error